src/components/controls/Dropdown.tsx

```diff
@@ -5,10 +5,15 @@ interface IDropdownState {
     selectedValue?: string;
 }
 
+export interface IDropdownItem {
+    key: string;
+    value: string;
+}
+
 interface IDropdownProps {
     label?: string;
     selected: string;
-    items: string[];
+    items: string[] | IDropdownItem[];
```

Comment:
Is your idea to migrate out of string[]? Would clean up the code in render quite a bit

Reply:
My idea is to allow Dropdown to be used in a simple mode, and in an advanced mode.